src/task.c

```diff
@@ -1530,6 +1539,8 @@ jl_task_t *jl_init_root_task(jl_ptls_t ptls, void *stack_lo, void *stack_hi)
     ct->reentrant_timing = 0;
     ct->reentrant_inference = 0;
     ct->inference_start_time = 0;
+    ct->last_scheduled_ns = 0;
+    ct->cpu_time_ns = 0;
```

Comment:
Wow, this PR was shockingly simple! 👌👌

We'll have to figure out all the details Jameson mentioned, but it looks great to me other than that!